app/Models/UserStatistics/Model.php

```diff
@@ -91,27 +91,27 @@ public function countryRank()
             ->count() + 1;
     }
 
-    public function __construct($attributes = [])
+    public function __construct($attributes = [], $null = false)
```

Comment:
@nanaya This is kinda dodgy - didn't want to break existing profile stuff. Thoughts on changing the profile stuff to do null->0 for display outside of this constructor?